cfme/tests/services/test_service_customer_bz.py

```diff
@@ -163,3 +165,29 @@ def test_update_dynamic_checkbox_field(appliance, import_datastore, import_data,
             "checkbox").checkbox.read() is True and not view.submit_button.disabled, timeout=60)
     except TimedOutError:
         pytest.fail("Checkbox did not checked and Submit button did not enable in time")
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(automates=[BZ(1720617)])
+@pytest.mark.customer_scenario
+@pytest.mark.parametrize("file_name", ["bz_1720617.yml"],
+    ids=["sample_dialog"],
+)
+def test_edit_import_dialog(appliance, import_dialog):
+    """Tests update import dialog.
+
+    Bugzilla:
+        1720617
+
+    Polarion:
+        assignee: nansari
+        casecomponent: Services
+        initialEstimate: 1/16h
+        tags: service
+    """
+    sd, ele_label = import_dialog
+    description = fauxfactory.gen_alphanumeric()
+    with update(sd):
```

Comment:
**Question:**
@niyazRedhat If you're modifying `description` of the imported dialog, `assert` is not required.?